### PR TITLE
feat: Change menu item title  name with editorItemTitle

### PR DESCRIPTION
**Pull request by iago1501**
#### What problem is this solving?

Menu Items created on site editor with the same name as "item"

image

Added the user's possibility to change this with __editorItemTitle field 

image

#### How to test it?

Go to the [Workspace](https://itemtitle--storecomponents.myvtex.com/admin/cms/site-editor)

Try to add a new menu item here: 

image

image

Fill Menu Item Name's field, go back, and check if the menu item name created is the same

image

#### Related to / Depends on

Zendesk KI: #342030

### CODE REVIEW

File: messages/en.json [hunk 1]
```diff
@@ -34,6 +34,8 @@
   "admin/editor.menu.item.category.title": "Category",
   "admin/editor.menu.item.custom.title": "Custom",
   "admin/editor.menu.item.iconId.title": "Icon ID",
+  "admin/editor.menu.item.editorItemTitle.title": "Menu Item Name",
+  "admin/editor.menu.item.editorItemTitle.description": "This will only be visible on site editor",
```
Comment (klzns) [inline messages/en.json:38]:
```suggestion
  "admin/editor.menu.item.editorItemTitle.description": "This will only be visible on Site Editor",
```

**Review by klzns — APPROVED** [review]

**Review by clara-cruz — CHANGES_REQUESTED** [review]
For ES, I suggest:
  "admin/editor.menu.item.editorItemTitle.title": "Nombre del Elemento del menú",
  "admin/editor.menu.item.editorItemTitle.description": "Esto solo será visible en Site Editor",

**Comment by iago1501** [comment]
> For ES, I suggest:
> "admin/editor.menu.item.editorItemTitle.title": "Nombre del Elemento del menú",
> "admin/editor.menu.item.editorItemTitle.description": "Esto solo será visible en Site Editor",

Did it ✌🏻

**Review by clara-cruz — APPROVED** [review]

**Comment by iago1501** [comment]
@vtex-apps/localization can you help me here with Japanese and also Romanian, please? 🙏🏻

**Review by camilavcoutinho — APPROVED** [review]
All Good! RO and JP ✨

**Comment by RamonSouzaRibeiro** [comment]
Great feature!
But I have a question, when I have some item with "ui:widget": "image-uploader", the image overwrite the title.

Example:
image

Is there something I can do to use the __editorItemTitle in this cases?

**Comment by martinlariaspaz** [comment]
> Great feature! But I have a question, when I have some item with "ui:widget": "image-uploader", the image overwrite the title.
> 
> Example: image
> 
> Is there something I can do to use the __editorItemTitle in this cases?

I need to know this too.

**Pull request merged**